_documentation/developers.md

```diff
@@ -40,14 +44,22 @@ composer install
 
 Kimai uses a SQLite database by default, which will work out-of-the-box. But you have to change your 
 environment to `dev` in your `.env` file. You can also configure a MySQL database if you prefer that:
+
 ```
 APP_ENV=dev
 DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name
 ```
 
+Run the Kimai installer:
+
+```bash
+bin/console kimai:install -n
```

Comment:
This is actually not needed. It was a bug in the reset-dev command in 1.10, which was fixed in https://github.com/kevinpapst/kimai2/pull/1982

If you use master branch, all works as expected.

We can keep it here in the meantime, but I bet I will forget to remove it with 1.11 ... can you remember to revert it 😁 ?

Reply:
I was installing api-user-now branch, I will remove this